sdk/src/wallet/bindings/java/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 group 'org.iota'
-version = '1.0.0-rc.1'
+version = '1.0.0-rc.3'
```

Comment:
Was RC2 ever released? Why the jump? If it was never released, can you make it RC2 and edit the changelog?

Reply:
no never, but i figured we align them anyways as we will merge them. ID rather not spend more time on this, as it has 0 uses on maven central anyways

Reply:
Yeah sure, just weird since you are going to release it, but whatever